### PR TITLE
[RUM] added documentation for automatic collection of user actions

### PR DESCRIPTION
**Pull request by hdelaby**
### What does this PR do?

Add documentation around the automatic collection of User Actions.

### Motivation

New feature.

### Preview link

https://docs-staging.datadoghq.com/hdelaby/automatic-collection-ua/real_user_monitoring/data_collected/user_action/

### CODE REVIEW

File: content/en/real_user_monitoring/data_collected/user_action.md [hunk 1]
```diff
@@ -16,16 +16,42 @@ further_reading:
...
+| `user_action.measures.long_task_count`        | number      | Count of all long tasks collected for this user action. |
+| `user_action.measures.resource_count`         | number      | Count of all resources collected for this user action. |
+| `user_action.measures.user_action_count`      | number      | Count of all user actions collected for this user action.|
 
 ## Facet Collected
 
 | Attribute    | Type   | Description              |
 |--------------|--------|--------------------------|
-| `event.name` | string | Name of the user action. |
+| `user_action.id` | string | UUID of the user action. |
```
Comment (bcaudan) [inline content/en/real_user_monitoring/data_collected/user_action.md:45]:
```suggestion
| `user_action.id` | string | UUID of the user action (not set for [Custom User Actions][5]). |
```

File: content/en/real_user_monitoring/installation/_index.md [hunk 2]
```diff
@@ -115,10 +115,11 @@ Paste the generated code snippet into the head tag (in front of any other script
 | `service`            | String  | No       | `` | The service name for this application.                             |
 | `env`                | String  | No       | `` | The application’s environment e.g. prod, pre-prod, staging.                   |
 | `version`            | String  | No       | `` | The application’s version e.g. 1.2.3, 6c44da20, 2020.02.13.                   |
+| `trackInteractions`          | Boolean  | No       | `false` | Enables [automatic collection of users actions][6] |
```
Comment (bcaudan) [inline content/en/real_user_monitoring/installation/_index.md:118]:
```suggestion
| `trackInteractions`          | Boolean  | No       | `false` | Enables [automatic collection of Users Actions][6] |
```
consistency

File: content/en/real_user_monitoring/data_collected/user_action.md [hunk 3]
```diff
@@ -16,16 +16,42 @@ further_reading:
   text: "Datadog Standard Attributes"
 ---
 
-A custom user actions is a custom event generated for a given user action. [Add one by instrumenting your code][1].
+## Automatic Collection of User Actions
+Real User Monitoring (RUM) SDKs automatically detect user interactions performed during a user journey. Set the `trackInteractions` [initialization parameter][1] to `true` to enable this feature.
```
Comment (bcaudan) [inline content/en/real_user_monitoring/data_collected/user_action.md:20]: It could be nice to be consistent by using either `interaction` or `user action`

**Comment by hdelaby** [comment]
Ready to merge :)

**Pull request merged**